frontend/src/components/team/CardList.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Card from "./Card";
+
+function CardList() {
+  const cardInfo = [
```

Comment:
Please move all kinds of data in a specific folder (ex: data) in your project

Reply:
Okay okay Mister @ioayoub

Reply:
You can double check